### DISCUSSION TITLE
Weak partial order implementation

### DISCUSSION THREAD

**Discussion by arthaud** [NASA-SW-VnV/ikos#281] [Ideas]
See https://github.com/facebookincubator/SPARTA/pull/5#issuecomment-566288425, that could impact our implementation too.

**Comment by ivanperez-keera** [comment]
@arthaud Nothing was ever done on this. This is actionable? Do you still think this is high priority?

**Comment by arthaud** [comment]
I think we would need to port https://github.com/facebook/SPARTA/pull/7 into IKOS.
It doesn't seem high priority though since we haven't noticed any concrete problems in IKOS's fixpoint.

**Comment by ivanperez-keera** [comment]
Ok. I'm going to move it to discussions as an idea, just to keep the issue list for items whose progress we are actively tracking. We can move it back if we start working on this in the future.

Thanks!
